CanDIG/CanDIGv2
# v4.0.0: Intervals Support, Service Tokens

# CanDIG & Integration tests:
- Support for date intervals
- Support for service tokens, rework of how auth works
- Added query integration tests

# HTSGet
https://github.com/CanDIG/htsget_app/releases/tag/v4.0.0
- Indexing changed to asynchronous task
- Adds service token support for Query
- Switched to hg38 for integration tests
- Query beacon results only return authorized results now

# Katsu
https://github.com/CanDIG/katsu/releases/tag/v4.0.0
- Date intervals change
- Explorer API
- Katsu's convert.py script can specify the name for datasets

# Data Portal
https://github.com/CanDIG/candig-data-portal/releases/tag/v3.0.0
- Footer added
- Completeness stats on summary & on new tab
- Date Interval support

# Query
https://github.com/CanDIG/candigv2-query/releases/tag/v2.2.0
- Update versions of Alpine and Python and other dependencies
- Several fixes to make it work with the new frontend, including genomic search fixes, etc.
- Fix a miscount on donors contributing to age summary stats once per treatment
- Nullness protection

# OPA
https://github.com/CanDIG/candig-opa/releases/tag/v2.1.0
- Add a path for Opa to verify service tokens
- Add a health check
- Add HTSGet paths
